03_Modules/EVDriver/src/module/api.ts

```diff
@@ -61,10 +61,20 @@ export class EVDriverModuleApi extends AbstractModuleApi<EVDriverModule> impleme
     }
 
     @AsMessageEndpoint(CallAction.RequestStopTransaction, RequestStopTransactionRequestSchema)
-    requestStopTransaction(identifier: string, tenantId: string, request: RequestStopTransactionRequest, callbackUrl?: string): Promise<IMessageConfirmation> {
+    async requestStopTransaction(identifier: string, tenantId: string, request: RequestStopTransactionRequest, callbackUrl?: string): Promise<IMessageConfirmation> {
         return this._module.sendCall(identifier, tenantId, CallAction.RequestStopTransaction, request, callbackUrl);
     }
 
+    @AsMessageEndpoint(CallAction.CancelReservation, CancelReservationRequestSchema)
+    async requestCancelReservation(identifier: string, tenantId: string, request: CancelReservationRequest, callbackUrl?: string): Promise<IMessageConfirmation> {
```

Comment:
according to method name "convention" in other api files, the name would be
```suggestion
    async cancelReservation(identifier: string, tenantId: string, request: CancelReservationRequest, callbackUrl?: string): Promise<IMessageConfirmation> {
```